CareKitStore/CareKitStore/Structs/OCKScheduleElement.swift

```diff
@@ -132,6 +130,7 @@ public struct OCKScheduleElement: Codable, Equatable {
     public init(start: Date, end: Date?, interval: DateComponents, text: String? = nil,
                 targetValues: [OCKOutcomeValue] = [], duration: Duration = .hours(1)) {
 
+        let calendar = Calendar.current
```

Comment:
#689 adds `calendar` to be a required encoded/decoded property which will break any `OCKTask` that's currently stored on a server in JSON because previous versions of CareKit never contained this property.

~Making the property a `var` and setting a default value will allow older saved OCKTasks's to be decoded as well as new ones (preserving all new updates with backwards compatibility).~ The other way would be to write a custom decoder for `OCKSheduleElement`, but that's more work and I don't think it's necessary.

Decoding Calendar.current after a few minutes seemed to cause crashes after ~10 minutes when `makeValue` is called during the hydration of a OCKTask from CoreData. Checking to see if this latest commit fixes the crash.